scripts/icm-injection-scripts/inject-icm.sh

```diff
@@ -43,23 +49,23 @@ EOF
 while IFS='' read -r content_set;
 do
   if [ "${content_set}" != "" ]; then
-    jq --arg content_set "$content_set" '.content_sets += [$content_set]' content-sets.json > content-sets.json.tmp
-    mv content-sets.json.tmp content-sets.json
+    jq --arg content_set "$content_set" '.content_sets += [$content_set]' ${icm_output_location} > content-sets.json.tmp
+    mv content-sets.json.tmp ${icm_output_location}
   fi
 done <<< "$(
     jq -r '
         if .bomFormat == "CycloneDX" then
             .components[].purl
         else
             .packages[].externalRefs[]? | select(.referenceType == "purl") | .referenceLocator
-        end' sbom-cachi2.json |
+        end' ${PREFETCH_SBOM} |
     grep -o -P '(?<=repository_id=).*(?=(&|$))' |
     sort -u
 )"
 
 echo "Constructed the following:"
-cat content-sets.json
+cat ${icm_output_location}
 
 echo "Appending a COPY command to the Containerfile"
 
-echo "COPY content-sets.json $location" >> "${CONTAINERFILE}"
+echo "COPY $icm_filename $location_in_container" >> "${CONTAINERFILE}"
```

Comment:
If I understand correctly, the `$icm_output_location` be at the root of the context directory? Otherwise the `$icm_filename` will not exist during the build? Which means the `$OUTPUT_DIRECTORY` must be the context directory?

Reply:
yes, that is correct.